lib/app/AppSettings.js

```diff
@@ -12,6 +12,10 @@ const logger = require('../logger')
 const fsEx = require('fs-extra')
 const { SETTINGS_FOLDER } = require('./Constants')
 
+/**
+ * @class AppSettings
+ * @implements {AppSettings}
```

Comment:
This is marked as "cyclic inheritance" because interface and class are named the same. I guess it would make sense to rename the interface.